e2e/testsuite/grpc_query.go

```diff
@@ -269,7 +275,13 @@ func (s *E2ETestSuite) GetBlockByHeight(ctx context.Context, chain ibc.Chain, he
 		return nil, err
 	}
 
-	return res.Block, nil
+	// versions newer than 0.47 SDK have the SdkBlock field.
+	if res.SdkBlock != nil {
+		return &res.SdkBlock.Header, nil
+	}
+
+	// older versions of the SDK do not have the SdkBlock field.
+	return &res.Block.Header, nil
```

Comment:
should we note somewhere that once support for v6 stops, this could be removed (maybe the interface as well?)?

Reply:
Agree! Could we create an issue and add a link to it in a code comment somewhere around here?

Reply:
will do!